### DISCUSSION TITLE
Can i change default styling even though i use the remote theme?

### DISCUSSION THREAD

**Discussion by ChangHyun2** [mmistakes/minimal-mistakes#2507] [Q&A] [answered]
> hello, i really like this minimal mistkae theme, and i'm using it to run my blog.
> thank you so much to make this theme.

and.. i have some questions,

> **1. should i use remote theme to make it work in github pages?**

I had tried to install this theme by using gem, and it worked locally very well. 
But when i pushed my website to remote repo, and checked in my github pages (ChangHyun2.github.io/blog) , it didn't work.
So, i searched some ways, and I found a way to solve this problem, such like travisCI.
i tried to use travis CI, but it was so difficult to me.. and i selected another way which is using a remote theme. 

**I wonder that should i use remote theme to work with github pages without using some tools like travisCI?**

> **2. Can i change default style by using remote theme?**

anway, i succeded to apply remote theme, and website runs well in locally and remotely in github pages. 
But, i want to change some default styles.
In documentation, it's stated to change repo folders to customize this theme.
But since i use a remote theme, there are only '_posts', '_site', 'vendor' folders in my website folder.

**How do i change default style with using a remote theme?**

**Comment by iBug** [comment, marked as answer]
Immediate duplicate of #2502

**Comment by iBug** [comment]
**Question 1**

If you don't use a CI service, then yes, you'll have to use the remote theme method.

Setting up CI isn't that complicated as you'd thought. [Here](https://github.com/ustclug/Linux101-docs/blob/master/.github/workflows/build.yml)'s a **GitHub Actions** config that I wrote. While that repository uses MkDocs, it should be easy enough to replace all the Python stuff with Ruby stuff (e.g. setup python → setup ruby, `pip install` → `bundle install`, `mkdocs build` → `bundle exec jekyll build`, etc.). The rest should remain more or less the same for your usage.

**Question 2**

See the above comment. There's the source of the duplicate for you.
